src/System.Net.Http/tests/FunctionalTests/HttpRequestMessageTest.cs

```diff
@@ -14,7 +14,7 @@ namespace System.Net.Http.Functional.Tests
 {
     public class HttpRequestMessageTest
     {
-        Version _expectedRequestMessageVersion = new Version(1, 1);
+        Version _expectedRequestMessageVersion = PlatformDetection.IsUap ? new Version(2,0) : new Version(1, 1);
```

Comment:
If we do that, it means we need SkipOnTargetFramework and/or Conditional* attributes on each test and having to rename the test methods with appropriate platform suffixes all the time.  We currently  don't use that pattern in other cases of platform differences like this.